src/dscom.build/IBuildContext.cs

```diff
@@ -61,4 +61,6 @@ public interface IBuildContext
     /// <param name="log">The log to write error messages to.</param>
     /// <returns><c>true</c>, if the conversion has taken place successfully; <c>false</c> otherwise.</returns>
     bool ConvertAssemblyToTypeLib(TypeLibConverterSettings settings, TaskLoggingHelper log);
+
+    bool EmbedTypeLib(TypeLibEmbedderSettings settings, TaskLoggingHelper log);
```

Comment:
Based on @carstencodes 's suggestions to use dscom client tools rather than calling dscom functions, I am not sure I want to invest more time in this. I considered removing the new classes but that would then create an inconsistency between the build task and the tool task. The removal of the build task should be a separate PR. Agree? 

Note that I will add more documentation to the dscom functions and to the dscom client.